cmd/openshift-install/create.go

```diff
@@ -328,7 +328,7 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 	discovery := client.Discovery()
 
 	apiTimeout := 20 * time.Minute
-	logrus.Infof("Waiting up to %v for the Kubernetes API at %s...", apiTimeout, config.Host)
+	logrus.Infof("Waiting up to %v from %v for the Kubernetes API at %s...", apiTimeout, config.Host, time.Now())
```

Comment:
I see the value in this. I suggest the following two changes.
1) Print the expiration time rather than the current time.
2) Use only the time and not the date. And limit the time to the minute.

Example,

> INFO Waiting up to 20m0s (until 5:25 PM) for the Kubernetes API at https://api.mhjacks-dc.blueprints.rhecoeng.com:6443...

Reply:
Sure!  Those make sense, and I will be happy to make those changes.